jest.config.js

```diff
@@ -1,3 +1,14 @@
 module.exports = {
-  testEnvironment: "node"
+  verbose: true,
+  watchPathIgnorePatterns: [
```

Comment:
Lista de testes que estão sendo ignorados e precisamos arrumar. Muitos desses testes ainda não estão refatorados e precisa de mock em algum deles por exemplo para funções de acesso a api do rocket e bot. 

List of tests that are being ignored and we need to clean up. Many of these tests are not yet refactored and need mock in some of them eg for access functions to the rocket and bot API.

Reply:
@cenicoemir